scripts/abi_to_rc/abi_to_rc.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+# By: Jon-Eric Cook
+# Github: @joneric
+
+from string import Template
+import argparse
+import json
+import sys
+import os
+import re
+
+# purpose of abi_to_rc.py
+purpose =   '''purpose:\n  The abi_to_rc.py script processes a contract's .abi file in order to generate 
+  an overview Ricardian Contract and a Ricardian Contract for each action. The 
+  overview Ricardian Contract provides a description of the contract's purpose 
+  and also specifies the contract's action(s), input(s), and input type(s). The 
+  action Ricardian Contract provides a description of the action's purpose and 
+  also specifies the action's input(s), and input type(s).'''
+
+## how to run abi_to_rc.py
+how_to =    '''\n \nhow to run:\n  $ python abi_to_rc.py /path/to/smart-contract.abi'''
+
+## example of how to run abi_to_rc.py
+example =   '''\n \nexample:\n  $ python abi_to_rc.py ../../contracts/currency/currency.abi'''
+
+# argument parser
+parser = argparse.ArgumentParser(
+    prog="abi_to_rc.py",
+    formatter_class=argparse.RawDescriptionHelpFormatter,
```

Comment:
I don't entirely approve of the usage of the RawDescriptionHelpFormatter.  If you `chmod +x abi_to_rc.py` it can be run the way the default formatter says, since you have a shebang at the beginning.  (And git will preserve the executable permission when people clone/pull.)  Long descriptions beg for automated line break formatting.  You can add the example to the `epilog` argument of ArgumentParser's constructor.  I don't insist, but try it and see.